### PR TITLE
Peninsula feature

### PR DESCRIPTION
**Pull request by getjump**
Feature that we deserved, it does measure length of your telegram ID!
Also it can display lowest (best) lengths out there! 🍆🍆🍆🍆

### CODE REVIEW

**Review by egregors — CHANGES_REQUESTED** [review]
@getjump gut gemacht, as always! 🔥 🔥 Aber could you please check a few of my improvement proposals?

File: bot/skills/length.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    def __init__(self, db_name: str):
+        self._coll: Collection = get_db(db_name).peninsulas
+
+    def get_longest_peninsula(self) -> int:
+        result = PENINSULA_MAXIMAL_LENGTH
+        x: PeninsulaDataType = self._coll.find_one(
+            sort=[(PENINSULA_LENGTH_FIELD, pymongo.DESCENDING)]
+        )
+        if x is not None:
+            result = max(result, int(x[PENINSULA_LENGTH_FIELD]))
```
Comment (C-Pro) [inline bot/skills/length.py:65]: Wouldn't this return 20 every time?

File: bot/skills/length.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        return result
+
+    def get_best_n(self, n: int = 10) -> List[PeninsulaDataType]:
+        return list(
+            self._coll.find({}).limit(n).sort(PENINSULA_LENGTH_FIELD, pymongo.ASCENDING)
```
Comment (C-Pro) [inline bot/skills/length.py:70]: Shouldn't sort come after limit?

File: bot/skills/length.py (hunk 1 above)
Comment (getjump) [reply bot/skills/length.py:65]: Yes, until telegram id would exceed 20 symbols. Actually that magic number was taken to limit eggplant amount to fit in one line on pc, we can probably tweak it smaller to fit on mobile.

File: bot/skills/length.py (hunk 2 above)
Comment (getjump) [reply bot/skills/length.py:70]: Yes, by common sense it is and that is a mistype. But actually in pymongo calling limit does nothing but sets limit in query (https://github.com/mongodb/mongo-python-driver/blob/c760f900f2e4109a247c2ffc8ad3549362007772/pymongo/cursor.py#L491), so query should work the same way no matter how you order a calls here.

**Review by egregors — APPROVED** [review]

**Pull request merged**